include/sdf/sdf.hh

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright (C) 2021 Open Source Robotics Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+#ifndef SDF_SDF_HH_
+#define SDF_SDF_HH_
+
+#pragma message("sdf/sdf.hh is deprecated, use sdformat.hh instead")
```

Comment:
we can decide if we want to deprecate `sdf/sdf.hh`; I'm not sure it's necessary

Reply:
reverted in e695e3c